cpp/unreal_projects/SpearSim/Python/validate_scene_kujiale.py

```diff
@@ -0,0 +1,591 @@
+import argparse
+import numpy as np
+import pathlib
+import posixpath
+import unreal
+import spear
+
+
+def validate_level(level):
+    #
+    # TODO:
+    # We name each scene using a lower_case_with_underscore naming convention and a four
+    # digit suffix (e.g., "apartment_0000", "apartment_0001", etc).
+    #
+
+    pass
+
+
+def validate_content_browser(level):
+    #
+    # TODO:
+    # There should be exactly one umap file with the same name as the scene in the scene's
+    # Maps directory. Our code imposes this requirement to support loading scenes by name.
+    #
+
+    #
+    # TODO:
+    # The "Content Browser" should not contain unreferenced assets, except for in the
+    # "Debug" directory.
+    #
+
+    pass
+
+
+def validate_outliner(level, iterate_all_actors):
```

Comment:
I think it would be cleaner simply to define `args` as a global variable at the top of this file. If we start passing around all possible future configuration options as function arguments throughout this function hierarchy, it wouldn't take very many configuration options for this approach to get messy.